cirq/qis/states_test.py

```diff
@@ -36,46 +37,69 @@ def assert_valid_density_matrix(matrix, num_qubits=None, qid_shape=None):
                                      dtype=matrix.dtype), matrix)
 
 
-def test_bloch_vector_simple_H_zero():
-    sqrt = np.sqrt(0.5)
-    H_state = np.array([sqrt, sqrt])
+@pytest.mark.parametrize('global_phase', (1, 1j, np.exp(1j)))
+def test_bloch_vector_zero_state(global_phase):
+    zero_state = global_phase * np.array([1, 0])
 
-    bloch = cirq.bloch_vector_from_state_vector(H_state, 0)
-    desired_simple = np.array([1, 0, 0])
+    bloch = cirq.bloch_vector_from_state_vector(zero_state, 0)
```

Comment:
What's the deal with changing the test here?  Is there a good reason from going from H state to zero_state?

Reply:
I reworded these because these names are very non-standard.  Quantum computing folk should know |0>,|1>,|+>,|->,|+i>,|-i>.   Also they violated style.